lib/modules/dosomething/dosomething_campaign/includes/CampaignController.php

```diff
@@ -2,6 +2,15 @@
 
 class CampaignController {
 
-  // @TODO: handle main app controller-y things such as CRUD in here.
+  /**
+   * Update the specified Campaign in storage.
+   *
+   * @param  int  $id
+   * @return void
+   */
+  function update($id) {
+    cache_clear_all('campaigns_api_request', 'cache_dosomething_api', TRUE);
```

Comment:
Well, only clears anything that begins with `campaigns_api_request*`, and rest is wildcard...

Yeah, I wasn't too keen on this, but need to keep in mind a specific campaign might be in a variety of different API related calls based on the query params...

`/campaigns?ids=362` <-- cached result
`/campaigns?ids=362,1321` <--- diff cached result
`/campaigns?staff_pick=true` <-- diff cached result that might include 362